tests/src/Simple/PInvoke/PInvoke.cs

```diff
@@ -118,8 +120,9 @@ private static void TestSafeHandle()
 
             ThrowIfNotEquals(true, SafeHandleTest(hnd, intVal), "Ansi String marshalling failed.");
         }
+#endif        
     }
-
+#if Windows_NT
```

Comment:
You will need to add the shims for errno to \corert\src\Native\System.Private.CoreLib.Native to make this work; and also delete stubbed out implementation of GetLastError from src\Native\Bootstrap\platform.unix.cpp